### PR TITLE
DOC: Update Specviz Plot Options documentation

### LINKED ISSUE #847: Add Documentation on loading Jdaviz uncertainties
As asked in https://github.com/spacetelescope/jdaviz/pull/832#issuecomment-914581684 . @javerbukh said something about `myinstance.app.get_viewer("spectrum-viewer").show_uncertainties()`. This needs to be documented.

Also see #839 .

[🐱](https://jira.stsci.edu/browse/JDAT-1742)

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to update Specviz Plot Options documentation and also show user how to show uncertainties via API.

Fixes #847

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/specviz/displaying.rst [hunk 1]
```diff
@@ -198,13 +198,25 @@ Line Width
...
 
-Minimum and maximum values of the Y axis.
+Opacity of the line. Maximum (1) is fully opaque and minimum (0) is fully transparent.
 
-Percentile
-----------
+Plot profile as steps
+---------------------
+
+Toggle on to view the spectrum as continuous line or step function.
```
Comment (ojustino) [inline docs/specviz/displaying.rst:209]:
```suggestion
Toggle on to view the spectrum as a continuous line or a step function.
```

**Review by ojustino — COMMENTED** [review]
I'm good with a merge pending a small change.

File: docs/specviz/displaying.rst (hunk 1 above)
Comment (pllim) [reply docs/specviz/displaying.rst]: Applied. Thanks!

**Review by ojustino — APPROVED** [review]

**Comment by pllim** [comment]
Thanks for the reviews!

**Pull request merged**